web-client/app/scripts/controllers/editProfileController.js

```diff
@@ -37,5 +40,96 @@ angular.module('webClientApp')
       $scope.error = 'حدث خطأ ما.'; // General form error.
       $scope.errorMessages = response.errors; // Detailed error message from backend.
     };
+    
+    /**
+     * Loads all user links
+     */
+    $scope.loadLinks = function() {
+      $scope.links = [{ loading: true }, { loading: true },
```

Comment:
Sorry I should've been more specific, initializing the links with loading obejcts.